tls/s2n_cipher_suites.c

```diff
@@ -1074,6 +1074,33 @@ int s2n_cipher_suites_cleanup(void)
     return 0;
 }
 
+S2N_RESULT s2n_iana_to_cipher_suite(const uint8_t iana[], struct s2n_cipher_suite **cipher_suite)
+{
+    ENSURE_REF(cipher_suite);
+    *cipher_suite = NULL;
+    ENSURE_REF(iana);
```

Comment:
Nit: Can we keep the safety checks together prior to initializing cipher_suite to NULL here?

Reply:
We could, but I'd prefer to set cipher_suite to NULL immediately. This way, even if the method fails because a null iana was passed in, we've set cipher_suite to NULL. That means that we can guarantee cipher suite is null if the method fails. It's not really a requirement, but it's nice and I think worth more than grouping the ENSURE_REFs is worth.